queue/util.lua

```diff
@@ -0,0 +1,41 @@
+local fiber = require('fiber')
+
+-- MAX_TIMEOUT == 100 years
+local MAX_TIMEOUT      = 365 * 86400 * 100
+-- Set to TIMEOUT_INFINITY
+-- instead returns time for next event
+local TIMEOUT_INFINITY = 18446744073709551615ULL
```

Comment:
What is the difference?

Reply:
MAX_TIMEOUT < TIMEOUT_INFINITY )
I understand you question, but in this task I only moved the common code to a separate place (DRY). It can be changed by a separate PR. I don't see any reasons to change it here.